Sources/MiamNeutraliOSFramework/ui/RecipeDetails/MiamNeutralRecipeDetailsTagsView.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  File.swift
+//  
+//
+//  Created by Damien Walerowicz on 03/01/2024.
+//
+
+import SwiftUI
+import MiamIOSFramework
+import miamCore
+
+@available(iOS 14, *)
+public struct MiamNeutralRecipeDetailsTagsView: RecipeDetailsTagsProtocol {
+    
+    public init() {}
+    public func content(
+        tags: [RecipeDetailTags]
+    ) -> some View {
+        WrappedHStack {
+            EmptyView()
+            ForEach(tags, id: \.id) { tag in
+                TagItem {
+                    HStack{
+                        Image.miamImage(icon: MiamIOSFramework.MiamIcon(rawValue: tag.picto)!)
```

Comment:
not sure what this is? 
we should either use .mealzIcons for a mealz image ( like Image.mealzIcon(icon: .pan) ) or use AsyncImage if its from the internet. 
clients can overwrite our icons in their own app

Reply:
I made changes to make it more readable